x-pack/plugins/apm/e2e/cypress/support/step_definitions/csm/service_name_filter.ts

```diff
@@ -8,16 +8,16 @@ import { When, Then } from 'cypress-cucumber-preprocessor/steps';
 import { DEFAULT_TIMEOUT } from '../apm';
 import { verifyClientMetrics } from './client_metrics_helper';
 
-When('a user changes the selected service name', (filterName) => {
+When('the user changes the selected service name', (filterName) => {
   // wait for all loading to finish
   cy.get('kbnLoadingIndicator').should('not.be.visible');
   cy.get(`[data-cy=serviceNameFilter]`, { timeout: DEFAULT_TIMEOUT }).select(
-    'opbean-client-rum'
+    'client'
   );
 });
 
 Then(`it displays relevant client metrics`, () => {
-  const metrics = ['0.01 sec', '0.07 sec', '7 '];
+  const metrics = ['4 ms', '0.06 s', '55 '];
```

Comment:
As someone unfamiliar with this test, Why did the metrics change?

Reply:
service name is changed that's selected on page load , it changes in above if you see from opbean-client-rum to clien.